### DISCUSSION TITLE
ذخیره نشدن Claim

### DISCUSSION THREAD

**Discussion by RohollahHosseini** [babaktaremi/Clean-Architecture-Template#11] [Q&A]
سلام خیلی ممنون  بابت این Repository 
من Identity  را هم اضافه کردم اما بعد از انجام عملیات لاگین برای من claim  ذخیره نمی شود ممنون میشم راهنمایی کنید

**Comment by babaktaremi** [comment]
توکن JWT بصورت Self Contained ساخته می شود. بهترین روش این هست که claim ها داخل توکن ذخیره شوند، برای اینکار میتونید کلاس زیر را با توجه به نیازتون تغییر بدید
https://github.com/babaktaremi/Clean-Architecture-Template/blob/master/src/Infrastructure/CleanArc.Infrastructure.Identity/Identity/AppUserClaimsPrincipleFactory.cs

**Comment by RohollahHosseini** [comment]
خیلی ممنون بابت راهنماییتون
